plugins/panel/plugin.js

```diff
@@ -361,7 +361,7 @@
 			 * @returns {*|CKEDITOR.dom.nodeList}
```

Comment:
This format is wrong. It will always return `CKEDITOR.dom.nodeList`, so please use this opportunity to adjust the API docs.